Gem/Code/Source/AutoGen/NetworkHealthComponent.AutoComponent.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+
+<Component
+    Name="NetworkHealthComponent" 
+    Namespace="MultiplayerSample" 
+    OverrideComponent="true" 
+    OverrideController="true" 
+    OverrideInclude="Source/Components/NetworkHealthComponent.h"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+    <NetworkProperty Type="float" Name="Health" Init="100.0f" ReplicateFrom="Authority" ReplicateTo="Client" Container="Object" IsPublic="true" IsRewindable="true" IsPredictable="false" ExposeToEditor="true" ExposeToScript="false" GenerateEventBindings="true" Description="The amount of health" />
+
+    <ArchetypeProperty Type="float" Name="MaxHealth" Init="100.0f" ExposeToEditor="true" Description="Maximum health" />
```

Comment:
We probably want an RPC (invoke from server, handle on authority) that applies a delta to the current health value.. that's what the weapon component would likely invoke when applying damage to various entities hit by a blast.